algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/models.py

```diff
@@ -399,7 +408,9 @@ def forward(self,
             self.out_proj.bias,
             key_padding_mask if key_padding_mask is not None else attn_mask,
             need_weights,
-            average_attn_weights)
+            average_attn_weights,
+            1 if key_padding_mask is not None else
+            0 if attn_mask is not None else None)
```

Comment:
nit: can you pull this out into a named variable? the nested ternary statements are a bit hard to read :)

Reply:
Yes, that's true! But, I left them in because thats how they are written in the pytorch 1.13 codebase and it will be easier to compare and incorporate diffs in the future?